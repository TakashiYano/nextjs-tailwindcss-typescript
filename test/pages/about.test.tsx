/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import About from "src/pages/about.page";

describe("About page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<About />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
