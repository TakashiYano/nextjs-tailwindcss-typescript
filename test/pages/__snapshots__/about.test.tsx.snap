// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
<DocumentFragment>
  <header>
    <div
      class="bg-white dark:bg-gray-800"
    >
      <div
        class="flex items-center justify-between"
      >
        <h1
          class="text-gray-900 dark:text-white"
        >
          LOGO
        </h1>
        <button
          aria-label="Toggle Dark Mode"
          class="w-10 h-10 p-3 ml-5 bg-gray-200 rounded dark:bg-gray-800"
          type="button"
        >
          <svg
            class="w-4 h-4 text-gray-800 dark:text-gray-200"
            fill="currentColor"
            stroke="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M12 3v1m0 16v1m9-9h-1M4 12H3m15.364 6.364l-.707-.707M6.343 6.343l-.707-.707m12.728 0l-.707.707M6.343 17.657l-.707.707M16 12a4 4 0 11-8 0 4 4 0 018 0z"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
        </button>
      </div>
      <nav>
        <a
          class="inline-block p-3 dark:text-gray-100"
          href="/"
        >
          Home
        </a>
        <a
          class="inline-block p-3 dark:text-gray-100"
          href="/about"
        >
          About
        </a>
      </nav>
    </div>
  </header>
  <main
    class="bg-green-100 dark:bg-gray-700"
  >
    <h2
      class="dark:text-white"
    >
      About
    </h2>
  </main>
  <footer
    class="bg-white dark:bg-gray-800"
  >
    <small
      class="dark:text-white"
      lang="en"
    >
      Â© 20xx example
    </small>
  </footer>
</DocumentFragment>
`;
